src/app/categories/[slug]/page.tsx

```diff
@@ -25,7 +24,7 @@ export default function CategoryPage({ params: { slug } }: Props) {
     );
 }
 
-export async function generateStaticParams() {
+export function generateStaticParams() {
```

Comment:
# Reviewer Note
It does not has to be `async` if you are not doing any API call